### ISSUE TITLE
Fix the use of the #names in files for GitBook version

### ISSUE THREAD

**Issue by deavmi** [WaspRC/Python-for-humans#1]
Fix the use of the #names in files for GitBook version.

**Comment by Walkman100** [comment]
#names?

**Comment by deavmi** [comment]
Yeah. What happens is when I do `omments.md#single-line-comments` GitBook doens't show those in the sidebar. It greys them out.

**Comment by deavmi** [comment]
@deavmi I updated the comment

**Comment by Walkman100** [comment]
Sorry, I have no idea how GitBook works, can't help you there

**Comment by deavmi** [comment]
@Walkman100 Maybe I should contact them and ask for a fix

**Comment by deavmi** [comment]
@Walkman100 It seems like the best thing to do. :)

**Comment by deavmi** [comment]
I just contacted them and I shall wait for a reply.
